spec/controllers/student_identifier_types_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe StudentIdentifierTypesController, type: :controller do
+  include ActiveJob::TestHelper
+
+  let(:organization)  { GitHubFactory.create_owner_classroom_org }
+  let(:user)          { organization.users.first                 }
+  let(:student)       { GitHubFactory.create_classroom_student   }
+  let(:student_identifier_type) do
```

Comment:
Since this is pretty generic, what do you think about moving this into a factory?